client/src/style/components.css

```diff
@@ -0,0 +1,27 @@
+header{
+        background-image: 
+        linear-gradient(
+                rgba(3, 2, 2, 0.541), 
+                rgba(5, 4, 4, 0.589)
+              ),url("../images/pplmakeglasgow.jpg");
+        background-size: cover;
+        background-repeat: no-repeat;
+        min-height: 300px;
+        background-position: right bottom, left top;
+        background-size: 100% 600px;
+        color:white;
+        display: flex!important;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        
+}
+
+@media (max-width: 600px) { 
```

Comment:
no need to change the background image

Reply:
Also we follow a [mobile-first](https://www.uxpin.com/studio/blog/a-hands-on-guide-to-mobile-first-design/) so your default CSS should be for small screen. Then override the bigger screen with `@media (min-width...` - not the other way around.